server/download.go

```diff
@@ -161,6 +161,9 @@ func (b *blobDownload) run(ctx context.Context, requestURL *url.URL, opts *Regis
 					log.Printf("%s part %d attempt %d failed: %v, retrying", b.Digest[7:19], i, try, err)
 					continue
 				default:
+					if try > 0 {
+						log.Printf("%s part %d completed after %d retries", b.Digest[7:19], i, try)
+					}
```

Comment:
Is this log line necessary?

Reply:
I thought it may be helpful to be explicit to see when the resumes worked and how many retries we hit, but that can be implied by looking for other "retrying" messages in the log for the same chunk.  I'm fine removing it if you'd like to keep it less chatty in these failure modes.